model/shared/src/main/scala/explore/model/User.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+import cats.derived.*
+import cats.syntax.all.*
+import io.circe.Decoder
+import lucuma.core.model.OrcidId
+import lucuma.core.model.UserProfile
+import lucuma.sso.client.codec.userProfile.given
+
+// Case class for the 'pi' and `users` in the Program. These must be
+// actual users, not service users.
+case class User(
```

Comment:
Do we have something similar in core or sso already?

Reply:
No. Core has a trait for the various users, but not what we need here.